src/sast.ml

```diff
@@ -4,11 +4,13 @@ type sexpr = dtype * sx
 and sx = 
     SNumberLit of float
   | SBoolLit of bool
-  | SStringLit of string
   | SCharLit of char
+  | SStringLit of string
+  | SListList of sexpr list
```

Comment:
should this be `SListLit` instead of `SListList`?

Reply:
yessir, will fix